src/platform_impl/web/mod.rs

```diff
@@ -29,6 +29,10 @@ mod backend;
 
 #[cfg(feature = "stdweb")]
 #[path = "stdweb/mod.rs"]
+#[cfg_attr(
+    feature = "stdweb",
+    deprecated(since = "0.23.0", note = "Please migrate to web-sys over stdweb")
+)]
```

Comment:
Wouldn't it be easier to put this at the top of the backend module?

Reply:
🤷, I'm not sure it changes much either way.

Reply:
You wouldn't need the feature check? Just seems cleaner to me.

Reply:
Good point, thanks 👍